backend/clubs/views.py

```diff
@@ -4911,6 +4911,17 @@ def complete_checkout(self, request, *args, **kwargs):
             Cart.objects.prefetch_related("tickets"), owner=self.request.user
         )
 
+        # Guard against holds expiring before the capture context
+        tickets = cart.tickets.filter(holder=self.request.user, owner__isnull=True)
+        if tickets.count() != cart.tickets.count():
```

Comment:
Note: this assertion checks that all the tickets in the cart are held by the user, so someone is not allowed to add more tix to cart without checking out (I'm ok with this)